code/modules/telesci/gps.dm

```diff
@@ -10,19 +10,28 @@ var/list/GPS_list = list()
 	origin_tech = list(TECH_DATA = 2, TECH_ENGINEERING = 2)
 	var/gps_prefix = "COM"
 	var/gpstag = "COM0"
-	var/emped = 0
-	var/held_by = null
-	var/implanted_into = null
+	var/compass_color = "#193A7A"
+
+	var/emped = FALSE
+
+	var/mob/held_by = null
+	var/mob/implanted_into = null
```

Comment:
need to clean up both of these in Destroy

Reply:
already handled
![image](https://user-images.githubusercontent.com/22774890/116808631-29a40500-ab3a-11eb-8edc-0208e6c3d7b6.png)